spread.yaml

```diff
@@ -0,0 +1,28 @@
+project: classic-snap
+
+path: /home/spread
+
+backends:
+  qemu:
+    systems:
+      - ubuntu-16.04-64:
+          username: ubuntu
+          password: ubuntu
+      - ubuntu-18.04-64:
+          username: ubuntu
+          password: ubuntu
+
+suites:
+  tests/main/:
+    summary: Integration test for the classic snap
+    prepare: |
+      echo "Prepare image"
+      apt update
+      apt install -y snapd snapcraft
+      snap install core
+      echo "Build snap"
+      cd $SPREAD_PATH
+      snapcraft clean
+      snapcraft
```

Comment:
Do we need to pass some options for the snap to be built correctly on different systems? (like clean build)

Reply:
Mostly because spread will transfert the source dir so if this is run inside a working tree that was used for building/testing things mihght be unclean.